tests/protractor/e2e/contacts/add-new-health-center.js

```diff
@@ -0,0 +1,43 @@
+const commonElements = require('../../page-objects/common/common.po.js'),
+	contactPage = require('../../page-objects/contacts/contacts.po.js'),
+	helper = require('../../helper'),
+	utils = require('../../utils');
+
+describe('Add new health center tests : ', () => {
```

Comment:
You're still submitting badly formatted code - sort out your formatter or get a better text editor. We use 2 spaces for indentation, your code has 8! https://github.com/medic/medic-docs/blob/master/development/style-guide.md#indention